### ISSUE TITLE
Feature: Easier Web GUI

### ISSUE THREAD

**Issue by gedw99** [ConduitIO/conduit#1062]
### Feature description

At wanted to extend the gui quite a bit for some ideas as its very simple right now. Simple is good, but there are many more opportunities to make the gui provide more functionality.

But I mean no disrespect but it is so complex for no reason.

Run this. It's another way to do web gui using the htmx pattern with a decent golang template system.

https://github.com/joerdav/go-htmx-examples

Just go run . 

the templates are a joy to work with IMHO.

---

Why ?

- so much easier and cleaner.
- there is NO build.  Only the template need a "go generate ."
- fast.... 
- much easier for a golang team to work on the GUI.
- easy to update the GUI by pushing from the back. HTMX accepts push updates from the backend. You push html over the wire.
- standards: https://htmx.org/examples/
- htmx is very popular actually.  
- any backend or frontend person can easily work on it. there is very little to learn.

**Comment by hariso** [comment]
> At wanted to extend the gui quite a bit for some ideas **as its very simple right now**. Simple is good, but there are many more opportunities to make the gui provide more functionality.
> 
> But I mean no disrespect **but it is so complex for no reason**.

The two bolded parts are contradictory. I guess you meant it's simple feature-wise, but complex development-wise (internally)?

**Comment by gedw99** [comment]
hey @hariso 

its simple in terms of development but great in terms of features. 

just play with this and its really obvious how power this simple approach is IMHO:
https://htmx.org/examples/

I noticed you have a repo at https://github.com/ConduitIO/mx-ui-components that also is a storybook.

https://github.com/joerdav/go-htmx-examples imports https://github.com/a-h/templ which has storybook support.
https://github.com/a-h/templ/tree/main/storybook

**Comment by simonl2002** [comment]
Not going to tag this for a milestone yet because there needs to be more discussion here.

**Comment by gedw99** [comment]
> Not going to tag this for a milestone yet because there needs to be more discussion here.

yep. That's partly why i added the reddit link, so you can see what others think

**Comment by simonl2002** [comment]
conduit UI is being retired. closing. https://github.com/ConduitIO/conduit/issues/1911
